tutorials/Kling_Gupta_efficiency.ipynb

```diff
@@ -0,0 +1,392 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0950f056",
+   "metadata": {},
+   "source": [
+    "# Kling–Gupta efficiency (KGE)\n",
+    "In this notebook, we will explore the Kling-Gupta Efficiency (KGE) which is widely used to evaluate the performance of hydrological models. KGE is a performance metric that decomposes the error into three components:\n",
+    "correlation,  variability and bias.\n",
+    "\n",
+    "It is computed as:\n",
+    "\n",
+    "$\n",
+    "\\text{KGE} = 1 - \\sqrt{\\left[s_\\rho \\cdot (\\rho - 1)\\right]^2 + \\left[s_\\alpha \\cdot (\\alpha - 1)\\right]^2 + \\left[s_\\beta \\cdot (\\beta - 1)\\right]^2}\n",
+    "$\n",
```

Comment:
@durgals this still needs fixing in the notebook - I presume you will get to this in due course, but just thought I'd mention it in case you thought myself or @Steph-Chong would be the ones making the change. I think it's best you take care of it as you're familiar with the score and the mathematics of it.